src/NuGet.Clients/NuGet.CommandLine/Common/CommandLineRepositoryFactory.cs

```diff
@@ -31,7 +31,17 @@ public override IPackageRepository CreateRepository(string packageSource)
                             System.ConsoleColor.Green,
                             "{0} {1}", args.Request.Method, args.Request.RequestUri);
                     }
-                    string userAgent = HttpUtility.CreateUserAgentString(CommandLineConstants.UserAgent);
+
```

Comment:
Because `HttpUtility` is defined in NuGet.Core.dll, which I didn't want to change within the scope of this PR. 
I just found out about a `UserAgent` class within NuGet.Protocol.Core.Types, I'll see if I can leverage that one instead.